src/components/MediaPlayer/VideoJSPlayer.js

```diff
@@ -316,6 +318,7 @@ function VideoJSPlayer({
           type: 'switchItem',
         });
       } else {
+        console.log('handle ended with null');
```

Comment:
Did you intend to leave these `console.log` calls?

Reply:
No I didn't! Good catch :100: